package.json

```diff
@@ -29,12 +29,13 @@
 		"Kryal - Da-Real-Kryall",
```

Comment:
When you already change the contributors, Crispr isnt on the server anymore

Reply:
Oh... Thanks